resources/js/pages/Auth/Login.vue

```diff
@@ -1,56 +1,42 @@
-                Sign in to Servidor
+    <sui-grid is="themed-page" layout="login" textAlign="center" verticalAlign="middle">
+        <sui-grid-column>
+            <h2 is="sui-header">
+                <sui-icon name="server" /> Sign in to Servidor
             </h2>
-            <form class="ui form" @submit.prevent="login" method="POST" action="/login">
-                <div class="ui inverted stacked segment">
+            <sui-form @submit.prevent="login">
+                <sui-segment stacked inverted>
                     <sui-message negative v-if="error.msg"
-                        :header="error.title"
-                        :content="error.msg" />
+                        :header="error.title" :content="error.msg" />
 
-                    <div class="field">
-                        <div class="ui left inverted transparent icon input"
-                            type="email" placeholder="Email address">
-                            <input id="email" type="email" name="email"
-                                v-model="username" placeholder="E-mail address"
-                                required autofocus>
-                            <i class="user icon"></i>
-                        </div>
-                    </div>
-                    <div class="field">
-                        <div class="ui left inverted transparent icon input"
-                            type="password" placeholder="Password">
-                            <input id="password" type="password" name="password"
-                                v-model="password" required
-                                placeholder="Password" />
-                            <i class="lock icon"></i>
-                        </div>
-                    </div>
-                    <div class="field left aligned">
-                        <div class="ui toggle checkbox">
-                            <input type="checkbox" name="remember" id="remember">
-                            <label for="remember">Remember Me</label>
-                        </div>
-                    </div>
-                    <button class="ui positive fluid large button" type="submit">
-                        Login
-                    </button>
-                </div>
-            </form>
+                    <sui-form-field>
+                        <sui-input inverted transparent required type="email"
+                            icon="user" icon-position="left" v-model="username"
+                            placeholder="Email address" autofocus />
+                    </sui-form-field>
+                    <sui-form-field>
+                        <sui-input inverted transparent required type="password"
+                            icon="lock" icon-position="left" v-model="password"
+                            placeholder="Password" />
+                    </sui-form-field>
 
-            <div class="ui inverted message">
+                    <sui-form-field class="left aligned">
```

Comment:
Check whether this has a proper alignment property

Reply:
It does not. Maybe submit a PR to sui-vue that adds a `textAlign` prop to `sui-form-field` similar to the one on `sui-grid`.